nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestCSVExportAction.java

```diff
@@ -228,52 +230,53 @@ public void testDownloadCSV() throws Exception {
 
         BulkStatus status = bulkService.getStatus(command.getId());
         assertEquals(COMPLETED, status.getState());
-        assertEquals(10, status.getProcessed());
-        assertEquals(10, status.getTotal());
+        assertEquals(DOC_BY_LEVEL, status.getProcessed());
+        assertEquals(DOC_BY_LEVEL, status.getTotal());
 
         Path dir = Files.createTempDirectory(CSVExportAction.ACTION_NAME + "test" + System.currentTimeMillis());
         File testZip = new File(dir.toFile(), "test.zip");
-        FileOutputStream out = new FileOutputStream(testZip);
-        HttpServletRequest request = mock(HttpServletRequest.class);
-        when(request.getMethod()).thenReturn("GET");
-
-        HttpServletResponse response = mock(HttpServletResponse.class);
-        ServletOutputStream sos = new DummyServletOutputStream() {
-            @Override
-            public void write(int b) throws IOException {
-                out.write(b);
-            }
-        };
-        PrintWriter printWriter = new PrintWriter(sos);
-        when(response.getOutputStream()).thenReturn(sos);
-        when(response.getWriter()).thenReturn(printWriter);
-
-        String url = (String) status.getResult().get("url");
-        downloadService.handleDownload(request, response, null, url);
+        try (FileOutputStream out = new FileOutputStream(testZip)) {
```

Comment:
Can you make this change in a separate commit, I don't see something related to trash.